sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/aio/_metrics_advisor_client_async.py

```diff
@@ -184,18 +182,14 @@ async def add_feedback(self, feedback, **kwargs):
                 :dedent: 4
                 :caption: Add new feedback.
         """
-        error_map = {
-            401: ClientAuthenticationError
-        }
```

Comment:
this was added in the generated code so I removed from the convenience layer

Reply:
In fact it was me that submitted the request to add 401 into default error_map. ;)